client/lib/analytics/utils/refresh-country-code-cookie-gdpr.js

```diff
@@ -20,35 +20,34 @@ export default async function refreshCountryCodeCookieGdpr( signal = undefined )
 	}
 
 	if ( refreshRequest === null ) {
-		refreshRequest = requestGeoData( signal ).then( ( { countryCode, region } ) => {
-			setCountryCodeCookie( countryCode );
-			setRegionCookie( region );
-		} );
+		refreshRequest = requestGeoData( signal )
+			.then( ( { country_short, region } ) => {
+				setCountryCodeCookie( country_short );
+				// For some IP ranges we don't detect the region and the value returned by the `/geo` endpoint is `"-"`.
+				// In that case set the cookie to `unknown` This cannot happen for `country_short` because the `/geo`
+				// endpoint returns a 404 HTTP status when not even the country can be detected.
+				setRegionCookie( region === '-' ? 'unknown' : region );
```

Comment:
Was the dash behavior being handled previously? I'm wondering if it'd be better to stop assigning `'unknown'` altogether for both of these values, and let it pass through as the original `'-'` (or `undefined`)?

Reply:
No, I discovered the `-` special value only when playing with the server implementation and making queries into the `ip2location` SQL table to see what's actually there. Sometimes the country is `-` (unknown) but that's handled and the `/geo` request fails completely with 404. But the auxiliary fields like `region` or `city` are optional and not always known.

I'd rather keep assigning `'unknown'` because that's the existing behavior and I don't want to bear the risk of forgetting to change some `code === 'unknown'` comparison somewhere.